library/tomcat

```diff
@@ -4,65 +4,30 @@ Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
-Directory: 10.1/jdk21/openjdk-bookworm
-
-Tags: 10.1.13-jdk21-openjdk-slim-bookworm, 10.1-jdk21-openjdk-slim-bookworm, 10-jdk21-openjdk-slim-bookworm, jdk21-openjdk-slim-bookworm, 10.1.13-jdk21-openjdk-slim, 10.1-jdk21-openjdk-slim, 10-jdk21-openjdk-slim, jdk21-openjdk-slim
-Architectures: amd64, arm64v8
-GitCommit: 4505ae69aa030beba88a68fe410866ee375a8fc0
-Directory: 10.1/jdk21/openjdk-slim-bookworm
-
 Tags: 10.1.13-jdk17-temurin-jammy, 10.1-jdk17-temurin-jammy, 10-jdk17-temurin-jammy, jdk17-temurin-jammy, 10.1.13-jdk17-temurin, 10.1-jdk17-temurin, 10-jdk17-temurin, jdk17-temurin, 10.1.13-jdk17, 10.1-jdk17, 10-jdk17, jdk17, 10.1.13, 10.1, 10, latest
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 4505ae69aa030beba88a68fe410866ee375a8fc0
+GitCommit: a9121f8cc83be6562532daee9c00b5498cd3a466
 Directory: 10.1/jdk17/temurin-jammy
 
 Tags: 10.1.13-jre17-temurin-jammy, 10.1-jre17-temurin-jammy, 10-jre17-temurin-jammy, jre17-temurin-jammy, 10.1.13-jre17-temurin, 10.1-jre17-temurin, 10-jre17-temurin, jre17-temurin, 10.1.13-jre17, 10.1-jre17, 10-jre17, jre17
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 4505ae69aa030beba88a68fe410866ee375a8fc0
+GitCommit: a9121f8cc83be6562532daee9c00b5498cd3a466
 Directory: 10.1/jre17/temurin-jammy
 
 Tags: 10.1.13-jdk11-temurin-jammy, 10.1-jdk11-temurin-jammy, 10-jdk11-temurin-jammy, jdk11-temurin-jammy, 10.1.13-jdk11-temurin, 10.1-jdk11-temurin, 10-jdk11-temurin, jdk11-temurin, 10.1.13-jdk11, 10.1-jdk11, 10-jdk11, jdk11
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 4505ae69aa030beba88a68fe410866ee375a8fc0
+GitCommit: a9121f8cc83be6562532daee9c00b5498cd3a466
 Directory: 10.1/jdk11/temurin-jammy
 
 Tags: 10.1.13-jre11-temurin-jammy, 10.1-jre11-temurin-jammy, 10-jre11-temurin-jammy, jre11-temurin-jammy, 10.1.13-jre11-temurin, 10.1-jre11-temurin, 10-jre11-temurin, jre11-temurin, 10.1.13-jre11, 10.1-jre11, 10-jre11, jre11
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 4505ae69aa030beba88a68fe410866ee375a8fc0
+GitCommit: a9121f8cc83be6562532daee9c00b5498cd3a466
 Directory: 10.1/jre11/temurin-jammy
 
-Tags: 9.0.80-jdk21-openjdk-bookworm, 9.0-jdk21-openjdk-bookworm, 9-jdk21-openjdk-bookworm, 9.0.80-jdk21-openjdk, 9.0-jdk21-openjdk, 9-jdk21-openjdk
-Architectures: amd64, arm64v8
-GitCommit: ad1ce61765fcc0421e91cf8ee0370c79eeaa7d79
-Directory: 9.0/jdk21/openjdk-bookworm
-
-Tags: 9.0.80-jdk21-openjdk-bullseye, 9.0-jdk21-openjdk-bullseye, 9-jdk21-openjdk-bullseye
-Architectures: amd64, arm64v8
-GitCommit: ad1ce61765fcc0421e91cf8ee0370c79eeaa7d79
-Directory: 9.0/jdk21/openjdk-bullseye
-
-Tags: 9.0.80-jdk21-openjdk-slim-bookworm, 9.0-jdk21-openjdk-slim-bookworm, 9-jdk21-openjdk-slim-bookworm, 9.0.80-jdk21-openjdk-slim, 9.0-jdk21-openjdk-slim, 9-jdk21-openjdk-slim
+Tags: 9.0.80-jdk21-corretto-al2, 9.0-jdk21-corretto-al2, 9-jdk21-corretto-al2, 9.0.80-jdk21-corretto, 9.0-jdk21-corretto, 9-jdk21-corretto
```

Comment:
Should we consider moving to al2023 since al2 EOL is planned (well, was planned and then extended)?

Reply:
Yeah, I think we need to add it as an alternative (working on a PR), since without it we lose `tomcat:11` alpha images. Tomcat 11 requires openssl3 and jdk21 and no other images have this combination right now.